src/rules/pyupgrade/rules/remove_six_compat.rs

```diff
@@ -8,7 +8,7 @@ use crate::registry::{Diagnostic, RuleCode};
 use crate::source_code::{Generator, Locator, Stylist};
 use crate::violations;
 
-/// Return `true` if the `Expr` is a reference to `${module}.${any}`.
+/// Return `true` if the [`Expr`] is a reference to `${module}.${any}`.
 fn is_module_member(call_path: &[&str], module: &str) -> bool {
```

Comment:
nit: Which `Expr`? (I think this doc comment is outdated)

Reply:
Ah yeah. Definitely outdated.